CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 cmake_minimum_required(VERSION 3.27)
 project("Pokemon_Project" VERSION 999.0.0)
 set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_COMPILE_WARNING_AS_ERROR ON)
```

Comment:
This is to be sure that warnings will be solved and not left there for "the future people"?